README.md

```diff
@@ -14,11 +14,12 @@ Transparently use [Twitter Bootstrap 3][twbs3] with [CakePHP 3][cakephp].
 ## What's included?
 
 - FlashComponent + elements (types: error, info, success, warning)
-- FormHelper (types: vertical, inline, horizontal | inputs: text, textarea, select, checkbox, radio)
-- Widgets (button, textarea)
+- FormHelper (types: default, inline, horizontal)
```

Comment:
Should be "align" instead of "types". `type` is already an option of create() so can cause confusion.